recipes/tardigrade_error_tools/meta.yaml

```
@@ -0,0 +1,100 @@
+{% set name = "tardigrade_error_tools" %}
+{% set version = "0.9.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/UCBoulder/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  path: '..'
+  sha256: af55feef37f99979c3ccfb20c2f563b5e10175cb7599264b441c9608fdb68652
+
+build:
+  number: 0
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script:
+    - set -x
+    # Build and install c++ libraries
+    - cmake -S . -B build -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_LIBDIR=lib
+    - cmake --build build --target tardigrade_error_tools tardigrade_error_tools_PYTHON
+    - cmake --install build --prefix ${PREFIX}
+    # Build and install Conda package
+    - $PYTHON -m pip install build/src/python/dist/$PKG_NAME-$PKG_VERSION.tar.gz --no-deps --ignore-installed -v --no-build-isolation
+    - $PYTHON -m pip show $PKG_NAME
+  missing_dso_whitelist:
+      - /lib64/librt.so.1
+      - /lib64/libc.so.6
+      - /lib64/libpthread.so.0
+  skip: true  # [osx or win]
+
+requirements:
+  build:
+    - cython
+    - python {{ python }}
+    - breathe >=4.3.0
+    - sphinx >=3.0.4
+    - sphinx_rtd_theme >=0.4.3
+    - sphinxcontrib-bibtex
+    - setuptools_scm[toml] >=6.2
+  host:
+    - {{ compiler('cxx') }}
+    - libgcc {{ libstdcxx }}             # [linux]
+    - libstdcxx {{ libstdcxx }}          # [linux]
```

```suggestion
    - {{ compiler('cxx') }}
```
`libgcc` and `libstdcxx` do not need to be added to the host section explicitly.  Also please move `{{ compiler('cxx') }}` to the build section.

---

I've verified this works using the local build utility.